CFC_WebApp/tests/TestFilterTrips.py

```diff
@@ -0,0 +1,99 @@
+import unittest
+import json
+from utils import load_database_json, purge_database_json
+from main import tripManager
+from pymongo import MongoClient
+import logging
+from get_database import get_db, get_mode_db, get_section_db, get_trip_db
+import re
+# Needed to modify the pythonpath
+import sys
+import os
+from datetime import datetime, timedelta
+from dao.user import User
+from dao.client import Client
+import tests.common
+
+logging.basicConfig(level=logging.DEBUG)
+
+class TestFilterTrips(unittest.TestCase):
```

Comment:
Why don't I add the __init__.py file, and then move the filter tests back into the tripManager tests? That way we can avoid having to do all those imports in the future, but then have the tests in a more reasonable place.